### PR TITLE
downloaded cypress along with first tests in new branch to align hist…

### PR DESCRIPTION
**Pull request by vl538**
…ories with main branch

### CODE REVIEW

File: cypress/e2e/src.cy.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+describe('Correctly generates line chart', () => {
+  it('passes', () => {
+    cy.visit('/')
+
+    cy.findByRole("link", { name: "Line" }).click()
+    cy.url().should('eq', 'http://localhost:8080/line.html')
```
Comment (sorensri) [inline cypress/e2e/src.cy.js]: Use baseUrl here '/line.html'

File: cypress/e2e/src.cy.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+describe('Correctly generates line chart', () => {
```
Comment (sorensri) [inline cypress/e2e/src.cy.js:1]: Remove these outer describe portions and shift-tab everything down. Replace 'passes' with the description

**Review by sorensri — CHANGES_REQUESTED** [review]
It looks good, just need two changes with the src.cy.js file

**Comment by vl538** [comment]
fixed the issue in new pull request

**Review by sorensri — APPROVED** [review]
Looks good

**Pull request merged**